app/models/space.rb

```diff
@@ -67,6 +67,19 @@ def metrics
     end
   end
 
+  def readableIdsToIds
```

Comment:
maybe rename to ``metric_readable_ids_to_ids_map``.  We should eventually put this in another file for ``graph.rb``, this is getting messy.